executor/src/witgen/processor.rs

```diff
@@ -37,6 +40,57 @@ impl<'a, T: FieldElement> OuterQuery<'a, T> {
     }
 }
 
+pub fn make_copy_constraints<T: FieldElement>(
```

Comment:
Tries to detect if we're in the `secondary_machine_plonk` example and hard-codes the copy constraints. We could also remove this and mark the test as `should_fail`, maybe that's better?

Reply:
Can you at least create an issue to replace this by properly handling the pil part?

Reply:
Yes, especially when it's public.

Reply:
OK, removed it!